pkg/apis/openebs.io/v1alpha1/cstor_backup.go

```diff
@@ -49,6 +49,9 @@ type CStorBackupSpec struct {
 
 	// BackupDest is the remote address for backup transfer
 	BackupDest string `json:"backupDest"`
+
+	// LocalSnap is flag to enable local snapshot only
+	LocalSnap bool `json:"localSnap"`
```

Comment:
how about naming it as 'BackupType'.. value as 'local' or 'remote'.. and default can be taken as 'remote' (or)
are we trying to take advantage of default value of 'false' for boolean?

Reply:
q_ : should size of the volume be saved on the backup. Consider a case where a snapshot is created, volume is resized, clone is created from the snapshot. There probably needs to be validation that the clone pvc - capacity should be same as the capacity during the snapshot creation.

Reply:
@vishnuitta  yes, we are taking advantage of default value of `false` to support existing volumesnapshotlocation without any modification. So better to keep it as `LocalSnap`, if enabled local snapshot else cloud backup.
@kmova PVC will be created by velero and it will be same as source pvc size. This can be added as BDD to validate this. I have planned writing BDD later in next PR in velero-plugin.